### DISCUSSION TITLE
GraphQL error reported after running npm run dev

### DISCUSSION THREAD

**Discussion by Charrichar** [gatsbyjs/gatsby#35934] [Help]
Hi, I just recently started using gatsby for a blog project. I followed the sign up and instructions on the website, got a repo created with gatsby. I used npm run setup, followed by npm run dev, but there is an error reported with graphql and a query, "Cannot query field "raw" on type "contentfulBlogPostDescriptionTextNode".". There are 3 Unhandled Graphql errors reported. That one, and the same message "Cannot query field "raw" on type for "contentfulBlogPostBodyTextNode" and "contentfulBlogPostDescriptionTextNode". Can someone help with this?

**Comment by ElegantStack** [comment]
From your Contentful control panel, make sure you fill **at least one** of the description fields of your blog posts.

**Comment by benomatis** [comment]
Interestingly, if I replaced `raw` with `description`, it started to work. so this is how it will look:
```
description {
  description
}

**Comment by benomatis** [comment]
...but really, the field has to be Rich Text for `raw` to become available. So the solution is to change the field type (not easy, because field type cannot be changed in Contentful).

Source: https://www.gatsbyjs.com/blog/how-to-use-the-contentful-rich-text-field-with-gatsby/
